README.md

```diff
@@ -1,2 +1,203 @@
-# stellar-docs
-Documentation for Stellar
+# Stellar Documentation and API Reference
+
+Welcome to the official home repository for [Documentation][docs] and
+[API Reference][api] for the [Stellar Network][stellar].
+
+# Table of Contents
+
+- [Stellar Documentation and API Reference](#stellar-documentation-and-api-reference)
+- [Table of Contents](#table-of-contents)
+- [Contributing](#contributing)
+- [Quick Start](#quick-start)
+- [Repository Structure](#repository-structure)
+- [Using Markdown](#using-markdown)
+  - [Markdown Basics](#markdown-basics)
+  - [Custom Markdown](#custom-markdown)
+    - [Alert](#alert)
+    - [Code Example](#code-example)
+
+# Contributing
+
+Contributions are more than welcome! Thank you! 🎉
+
+Before diving in, please read our [Stellar Contribution Guide][contrib] for
+details on contributing to Stellar's various repositories. Please take special
+note of the code of conduct and contributor license agreement.
+
+Our documentation site is built using [Docusaurus][docusaurus]. The content is
+written in [MDX][mdx], which adds a lot of cool possibilities. Even if you're
+unfamiliar with plain markdown, do not fear! You can still contribute in a
+helpful and meaningful way.
+
+# Quick Start
+
+To begin development on the documentation, you will first need [yarn][yarn]
+installed on your system. Once it is, you can run the following commands:
+
+```bash
+git clone https://github.com/stellar/stellar-docs
+cd stellar-docs
+yarn install
+npx docusaurus start
+```
+
+This will begin the devlopment server, and open a browser window/tab pointing to
+`http://localhost:3000/docs/`. This development server will auto-reload when it
+detects changes to the repository.
+
+# Repository Structure
+
+- `/docs/` Contains all the documentation content. If you're contributing to the
+  actual documentation, rather than site functionality, this is likely where you
+  will want to be.
+  - `/docs/<subdirectory>/` Each subdirectory inside the `docs` directory
+    contains content documents relating to a common idea (asset issuance, or
+    core node administration, for example). There can also be subdirectories
```

Comment:
Hmm, is "core node administration" the right wording? I guess I've never heard of it said this way. Maybe "running a validator node" is better? I may be wrong, and if so, obviously feel free to ignore this.

Reply:
They both I _think_ communicate the same idea, but I like what you put much better. Sounds much like a much more natural phrase. 👍🏻